lib/cfa/base_model.rb

```diff
@@ -115,14 +128,20 @@ def modify(key, value)
       true
     end
 
+    # Replace a commented out entry and return `true`,
+    # or do nothing and return `false`.
+    # @return [Boolean]
     def uncomment(key, value)
       # Try to find if it is commented out, so we can replace line
       matcher = Matcher.new(
         collection:    "#comment",
+        # FIXME: this assumes a specific "=" syntax, bypassing the lens
```

Comment:
yes, it is just fallback uncomment, I expect that if needed, model can reimplement this method.